### ISSUE TITLE
train the syncnet

### ISSUE THREAD

**Issue by shz0519** [taewookim/keras-syncnet#2]
Hi,thank you for the excellent work and publicly available code.
have you trained the syncnet? I'm trying to use the mvlrs_v1 datasets to train the Syncnet,but the loss keep oscillating.
Since most video in the mvlrs_v1 is short,I randomly shift the audio up to 25-frame in order to generate synthetic false audio-video pairs.
1、Is each batch train data generated by one video or multiple videos ?
2、Does the false pairs have to be shifted by up to 2s?
3、Can you show me your script to generate the train datasets？
thank you!
